<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TmsTrendDialog</class>
 <widget class="QDialog" name="TmsTrendDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>580</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="btnDateTime">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>5</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="srview.qrc">
     <normaloff>:/images/time-planning.png</normaloff>:/images/time-planning.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>5</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="srview.qrc">
     <normaloff>:/images/right.png</normaloff>:/images/right.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnZoomIn">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>5</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="srview.qrc">
     <normaloff>:/images/zoomin.png</normaloff>:/images/zoomin.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnZoomOut">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>5</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="srview.qrc">
     <normaloff>:/images/zoomout.png</normaloff>:/images/zoomout.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>431</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TREND GROUP</string>
   </property>
  </widget>
  <widget class="QLabel" name="tag1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>T-N</string>
   </property>
  </widget>
  <widget class="QLabel" name="desc1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>총질소</string>
   </property>
  </widget>
  <widget class="QLabel" name="tag2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>T-P</string>
   </property>
  </widget>
  <widget class="QLabel" name="desc2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>총질소</string>
   </property>
  </widget>
  <widget class="QLabel" name="tag3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>COD</string>
   </property>
  </widget>
  <widget class="QLabel" name="desc3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>화학적산소요구량</string>
   </property>
  </widget>
  <widget class="QLabel" name="tag4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PH</string>
   </property>
  </widget>
  <widget class="QLabel" name="desc4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>수소이온농도</string>
   </property>
  </widget>
  <widget class="QLabel" name="tag6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BOD</string>
   </property>
  </widget>
  <widget class="QLabel" name="tag5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SS</string>
   </property>
  </widget>
  <widget class="QLabel" name="desc6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>생물확적산소요구량</string>
   </property>
  </widget>
  <widget class="QLabel" name="desc5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>부유물질량</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnTag1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>75</y>
     <width>50</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnTag2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>135</y>
     <width>50</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnTag3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>195</y>
     <width>50</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnTag4">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>255</y>
     <width>50</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnTag5">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>315</y>
     <width>50</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnTag6">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>375</y>
     <width>50</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="trendFrame">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>741</width>
     <height>521</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btnPrev">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>5</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="srview.qrc">
     <normaloff>:/images/left.png</normaloff>:/images/left.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnSelectAll">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>5</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="srview.qrc">
     <normaloff>:/images/select_all.png</normaloff>:/images/select_all.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>5</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="srview.qrc">
     <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>btnTag3</zorder>
  <zorder>btnTag2</zorder>
  <zorder>btnTag4</zorder>
  <zorder>btnTag5</zorder>
  <zorder>btnTag6</zorder>
  <zorder>btnTag1</zorder>
  <zorder>widget</zorder>
  <zorder>btnDateTime</zorder>
  <zorder>btnNext</zorder>
  <zorder>btnZoomIn</zorder>
  <zorder>btnZoomOut</zorder>
  <zorder>label</zorder>
  <zorder>tag1</zorder>
  <zorder>desc1</zorder>
  <zorder>tag2</zorder>
  <zorder>desc2</zorder>
  <zorder>tag3</zorder>
  <zorder>desc3</zorder>
  <zorder>tag4</zorder>
  <zorder>desc4</zorder>
  <zorder>tag6</zorder>
  <zorder>tag5</zorder>
  <zorder>desc6</zorder>
  <zorder>desc5</zorder>
  <zorder>trendFrame</zorder>
  <zorder>btnPrev</zorder>
  <zorder>btnSelectAll</zorder>
  <zorder>okButton</zorder>
 </widget>
 <resources>
  <include location="srview.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>TmsTrendDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>983</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
